src/content/guide/tools-and-features/remote-diagnostics.md

```diff
@@ -0,0 +1,320 @@
+---
+word: Remote Diagnostics
+title: Remote Diagnostics
+order: 11
+columns: two
+layout: guide.hbs
+devices: [photon, electron]
+---
+
+# {{title}}
+
+<p class="boxedHead">Beta feature</p>
+<p class="boxed">
+This feature is **currently in private beta**.
+If you would like access to this feature, please
+<a href="https://www.particle.io/sales" target="_blank">contact us</a>.
```

Comment:
Maybe "contact sales" instead of "contact us"? (I may remember it wrong, but I think we changed the phrasing a few months ago)

Reply:
Hmmm...I'm not sure. That's a good thing to clarify. I'll leave it for now but let's bring this up with the marketing/sales team